collectibles.go

```diff
@@ -1,6 +1,7 @@
 package blockatlas
 
 type Collection struct {
+	CollectionId    string `json:"collection_id"`
```

Comment:
can we just call id `id` ? it's already returning collection model

Reply:
In client used collection_id. In next api version good idea use id.

Reply:
which client? `v2/ethereum/collections/0x5574Cd97432cEd0D7Caf58ac3c4fEDB2061C98fB` does not have `collection_id `

Reply:
I looked /collectibles/category.

Reply:
`id` should be good to go then!